internal/quaiapi/quai_api.go

```diff
@@ -757,7 +768,9 @@ func (s *PublicBlockChainQuaiAPI) ReceiveWorkShare(ctx context.Context, raw json
 		txs, err := s.b.GetTxsFromBroadcastSet(workShare.TxHash())
 		if err != nil {
 			txs = types.Transactions{}
-			s.b.Logger().Warn("Failed to get txs from the broadcastSetCache", "err", err)
+			if len(txs) != 0 && workShare.TxHash() != types.EmptyRootHash {
```

Comment:
I think this check is wrong - `txs = types.Transactions{}` means `len(txs)` must be zero in every case

Reply:
yes, i need to remove the length check here, the objective of this is to eliminate the print when we dont have transaction

Reply:
done